zerobuf/share/zerobuf/CMake/zerobufGenerateCxx.cmake

```diff
@@ -19,9 +19,9 @@ function(zerobuf_generate_cxx Name OutputDir)
     list(APPEND ${Name}_SOURCES ${ZEROBUF_SOURCE})
 
     add_custom_command(
-      COMMAND ${ZEROBUF_BASE_DIR}/bin/zerobufCxx.py
+      COMMAND python ${ZEROBUF_BASE_DIR}/bin/zerobufCxx.py
```

Comment:
Depending on a platforms handling of concurrent installations of python 2 & 3, this might fail. I'd use the ${PYTHON_EXECUTABLE} CMake variable, so the selection of the python version can be controlled with the corresponding find_package() call.